### PR TITLE
Add Laravel to PHP

### LINKED ISSUE #1939: [Small Edit] PHP: add laravel
### Type of Edit (select all that apply)

Clarify unclear content and/or terms

### Description (optional)

This should go in this file `https://github.com/Codecademy/docs/content/php/php.md`

https://www.codecademy.com/resources/docs/php

As the most popular php framework we would like to have some note about laravel here 
I see symfony not laravel
if you are more enthusiastic add about codeIgniter too!

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by LucaGalassi**
### Description

Added Laravel framework to php
Closes #1939

### Type of Change
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist
- [x] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/php.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ A PHP interpreter makes a website dynamic and customizable, allowing a site to r
 
 [Symfony](https://symfony.com/at-a-glance), a French PHP framework provider, works on PHP-based projects for organizations of all kinds. This includes [the 2016 campaign site](https://symfony.com/blog/an-open-source-web-platform-for-the-new-president-of-france) of French President Emmanuel Macron.
 
-Lastly, [CakePHP](https://cakephp.org/) is an open-source framework that has been used by various companies, including:
+[Laravel](https://google.com) is another PHP framework used for developing web applications. It is known for its elegant syntax, modular structure, and built-in features that make it easier to develop high-quality web applications. Many leading companies use Laravel to develop their web applications including [GitLab](https://about.gitlab.com/) and [Pfizer](https://www.pfizer.com/).
```
Comment (SSwiniarski) [inline content/php/php.md:27]: Does Google have a Laravel page? Can we link to that?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@lucagalassi I did a first review and left a comment.

File: content/php/php.md (hunk 1 above)
Comment (lucagalassi) [reply content/php/php.md]: I totally missed that I didn't update the link. Thanks for pointing this out, it should be all good now.

**Review by SSwiniarski — APPROVED** [review]
@LucaGalassi Looks good for a second review.

**Pull request merged**